### PR TITLE
[DGPFUS-40] Querying the Transportation Database

### PR DESCRIPTION
**Pull request by Otruke**
DB passengers that did not survived
DB PidNAME100age35spouseSIBLINGSaboard
DB name and sex less than 27 embarkation at QandC
DB SURVIVED PASSENGERS GOING TO S
DB3 survived passengers

### CODE REVIEW

File: submissions/rufusOtolorin/database3/Querying the transportation Database.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+WHERE accident_cases.survivals = 1;
+-- second, to generate the count
+SELECT COUNT(*) FROM travel_details INNER JOIN accident_cases ON travel_details.passengers_id = accident_cases.passengers_id AND travel_details.embarkation = 'S' 
+WHERE accident_cases.survivals = 1;
+-- SURVIVED NUMBER OF PASSENGERS THAT EMBARKED AT SOUTHAMPTON IS 218
+
+/* (Q5). TO GET THE ID, NAME AND THE TOTAL NUMBER OF PASSENGERS WHO PAID
+A FARE GREATER THAN $100 AND ABOVE THE AGE OF 35 HAD SIBLINGS OR
+SPOUSES ON BOARD. */
+SELECT passengers.id, passengers.full_name FROM passengers INNER JOIN travel_details ON passengers.id = travel_details.passengers_id
```
Comment (Mayowa2020) [inline submissions/rufusOtolorin/database3/Querying the transportation Database.sql:26]: Well done @Otruke, please attach the list or simply the total number of passengers in this case.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
For question 5, please attach the list or simply state the total number of passengers involved in this case.

**Comment by Otruke** [comment]
UPDATE DONE
DB NO5 TOTALnumberIS9

File: submissions/rufusOtolorin/database3/Querying the transportation Database.sql (hunk 1 above)
Comment (Otruke) [reply submissions/rufusOtolorin/database3/Querying the transportation Database.sql:27]: UPDATE has been made

Comment (Otruke) [reply submissions/rufusOtolorin/database3/Querying the transportation Database.sql:27]: DB NO5 TOTALnumberIS9

**Review by Mayowa2020 — APPROVED** [review]
Well done @Otruke! Approved..

**Pull request merged**